modules/renter/hostdb/persist.go

```diff
@@ -66,6 +66,11 @@ func (hdb *HostDB) load() error {
 		if err != nil {
 			hdb.log.Debugln("ERROR: could not insert host while loading:", host.NetAddress)
 		}
+
+		// Make sure that all hosts have gone through the initial scanning.
+		if len(host.ScanHistory) < 2 {
+			hdb.queueScan(host)
+		}
```

Comment:
scan hosts if they haven't been scanned before. This makes sure that you don't miss out on good hosts if you shutdown siad too quickly after your first load.